core/modules/wmgr/python/dbMgr.py

```diff
@@ -316,36 +325,19 @@ def createTable(dbName):
         raise ExceptionResponse(400, "InvalidArgument",
                                 "`schemaSource` parameter has unexpected value \"%s\"" % schemaSource)
 
+    dbConn = Config.instance().dbConn()
+
     if schemaSource == 'request':
 
-        # get table name from schema
-        match = _createTableRe.match(schema)
-        if match is None:
-            raise ExceptionResponse(400, "InvalidArgument",
-                                    "`schema` parameter is not a CREATE TABLE statement")
-        if match.group(1):
-            raise ExceptionResponse(400, "InvalidArgument",
-                                    "CREATE TABLE statement includes database name")
-        schemaTblName = match.group(2)
-        _log.debug('table name from schema in request: %s', schemaTblName)
-        if tblName:
-            if schemaTblName != tblName:
-                _log.error('table name does not match schema table name')
-                raise ExceptionResponse(400, "InvalidArgument",
-                                        "Table name in `schema` does not match `table`: %s vs %s" %
-                                        (schemaTblName, tblName))
-        else:
-            tblName = schemaTblName
-            _validateTableName(tblName)
-
-        # need to drop CREATE TABLE part, everything before (
-        idx = schema.find('(')
-        if idx < 0:
-            _log.error('schema is missing opening parenthesis: %s', schema)
-            raise ExceptionResponse(400, "InvalidArgument",
-                                    "CREATE TABLE statement has no open parenthesis")
-        schema = schema[idx:]
-        _log.debug('schema from request: %s', schema)
+        # assume that we have correct SQL already
+        try:
+            dbConn.useDb(dbName)
+            dbConn.execCommand0(schema)
```

Comment:
Indeed, `execCommandN()` could be used for anything. You can think of `execCommand0()` is just an optimization on top of `execCommandN()`, the difference is that `execCommand0()` does not care about result returned from query, while `execCommandN()` returns full result as list of rows (and `execCommand1()` returns single row).